internal/cmd/cancel.go

```diff
@@ -0,0 +1,49 @@
+// Package cmd for handling commands
+package cmd
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog/log"
+	"github.com/threefoldtech/grid3-go/deployer"
+	"github.com/threefoldtech/grid3-go/internal/config"
+)
+
+// Cancel cancels a deployed project
+func Cancel(projectName string) error {
+	log.Info().Msgf("canceling contracts for project %s", projectName)
+	path, err := config.GetConfigPath()
+	if err != nil {
+		return errors.Wrap(err, "failed to get configuration file")
+	}
+
+	var cfg config.Config
+	err = cfg.Load(path)
+	if err != nil {
+		return errors.Wrap(err, "failed to load configuration try to login again using gridify login")
+	}
+	tfclient, err := deployer.NewTFPluginClient(cfg.Mnemonics, "sr25519", cfg.Network, "", "", "", true, false)
```

Comment:
lets move TFPluginClient to a struct and add deploy, cancel, .... to the struct methods to avoid creating it inside all functions